============================= test session starts ==============================
platform linux -- Python 3.7.6, pytest-4.6.2, py-1.8.0, pluggy-0.12.0 -- /home/tiago/codenation/data-science-2/venv/bin/python
cachedir: .pytest_cache
rootdir: /home/tiago/codenation/data-science-2
collecting ... collected 7 items

test_submit.py::test_q1 PASSED                                           [ 14%]
test_submit.py::test_q2 PASSED                                           [ 28%]
test_submit.py::test_q3 PASSED                                           [ 42%]
test_submit.py::test_q4 PASSED                                           [ 57%]
test_submit.py::test_q5 PASSED                                           [ 71%]
test_submit.py::test_q6 PASSED                                           [ 85%]
test_submit.py::test_q7 PASSED                                           [100%]

=============================== warnings summary ===============================
venv/lib/python3.7/site-packages/ipywidgets/widgets/widget_selection.py:9
venv/lib/python3.7/site-packages/ipywidgets/widgets/widget_selection.py:9
  /home/tiago/codenation/data-science-2/venv/lib/python3.7/site-packages/ipywidgets/widgets/widget_selection.py:9: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    from collections import Mapping, Iterable

venv/lib/python3.7/site-packages/numpy/lib/histograms.py:824
venv/lib/python3.7/site-packages/numpy/lib/histograms.py:824
  /home/tiago/codenation/data-science-2/venv/lib/python3.7/site-packages/numpy/lib/histograms.py:824: RuntimeWarning: invalid value encountered in greater_equal
    keep = (tmp_a >= first_edge)

venv/lib/python3.7/site-packages/numpy/lib/histograms.py:825
venv/lib/python3.7/site-packages/numpy/lib/histograms.py:825
  /home/tiago/codenation/data-science-2/venv/lib/python3.7/site-packages/numpy/lib/histograms.py:825: RuntimeWarning: invalid value encountered in less_equal
    keep &= (tmp_a <= last_edge)

venv/lib/python3.7/site-packages/statsmodels/nonparametric/kde.py:447
  /home/tiago/codenation/data-science-2/venv/lib/python3.7/site-packages/statsmodels/nonparametric/kde.py:447: RuntimeWarning: invalid value encountered in greater
    X = X[np.logical_and(X > clip[0], X < clip[1])] # won't work for two columns.

venv/lib/python3.7/site-packages/statsmodels/nonparametric/kde.py:447
  /home/tiago/codenation/data-science-2/venv/lib/python3.7/site-packages/statsmodels/nonparametric/kde.py:447: RuntimeWarning: invalid value encountered in less
    X = X[np.logical_and(X > clip[0], X < clip[1])] # won't work for two columns.

-- Docs: https://docs.pytest.org/en/latest/warnings.html
----- generated xml file: /home/tiago/codenation/data-science-2/output.xml -----
==================== 7 passed, 8 warnings in 11.62 seconds =====================
